src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -1095,47 +1100,88 @@ public TFTensor GetBufferedBatchTensor()
         }
     }
 
-    public sealed class TensorFlowEstimator : TrivialEstimator<TensorFlowTransform>
+    public sealed class TensorFlowEstimator : IEstimator<TensorFlowTransform>
     {
-        public TensorFlowEstimator(IHostEnvironment env, string model, string[] inputs, string[] outputs)
-           : this(env, new TensorFlowTransform(env, TensorFlowUtils.GetSession(env, model), inputs, outputs, TensorFlowUtils.IsSavedModel(env, model) ? model : null, false))
+        private readonly IHost _host;
+        private readonly TensorFlowTransform.Arguments _args;
```

Comment:
>TensorFlowTransform.Arguments _args [](start = 25, length = 35)

@Zruty0, are you ok with that? Or you ok with check this in, and clean it after?

Reply:
Let me know if this needs to be done in different way. I will update it.

---
In reply to: [227577377](https://github.com/dotnet/machinelearning/pull/1208#discussion_r227577377) [](ancestors = 227577377)